fixtures/jar-app/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM gcr.io/GCP_PROJECT/openjdk8:testing
+ADD spring-boot-sample-servlet-1.4.0.RELEASE.jar app.jar
+CMD [ "java","-Djava.security.egd=file:/dev/./urandom","-jar","/app.jar"]
```

Comment:
I copied this from the Dockerfile template that gcloud uses when deploying a jar to the openjdk8 runtime. It may very well not be necessary for our purposes.